penpost/penpost-web
# Jk/feature/postcard preview

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

Adds styling to the postcard preview form. Adds an edit button that passes the users postcard data back to the form page. Additionally adds minor styling to the postcard form page.

Issues closed #38 #21 #20

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

N/A

> Describe your code changes in detail for reviewers.

Update preview-page sass file to add styling. Add back button to preview page.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Postcard-Preview.js. _Postcard-Preview.scss. Postcard-From.js. _postcard-form.scss.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

In browser, ensure that site navigation is still functional, and styling is applied properly.

### Steps To Take
> Any questions and possible steps moving forward.

Implement graphQL.
